.github/workflows/ci-pipeline.yaml

```diff
@@ -17,6 +17,17 @@ jobs:
   get-chart:
     runs-on: ubuntu-latest
     name: 'Get modified charts'
+    # Given performance issues of the action feature on GH's side, we need to be very restrictive in the job's triggers:
+    # -> The 'Get modified charts' job suceededs AND
+    # -> The event is performed over an open PR  AND
+    #  ( ---> The pipeline was triggered due to a label addition and said label was the 'verify' one OR
+    #    ---> The pipeline was NOT triggered due to a label addition but the PR already contains the 'verify' one )
+    if: |
+        github.event.pull_request.state != 'closed' &&
+        (
+          (github.event.action == 'labeled' && github.event.label.name == 'verify') ||
+          (github.event.action != 'labeled' && contains(github.event.pull_request.labels.*.name, 'verify'))
+        )
```

Comment:
Although I understand the intentions of moving this conditional logic to the first job, I believe it's not the right move.

The _Get modified charts_ step should be executed regardless of the existence of the `verify` tag, as it serves as early feedback to the contributor. For instance, it reports whether the expected chart version bump exists or warns when modifications to multiple assets occur:

https://github.com/bitnami/charts/blob/97a1cba706cf292079bd800c8bc582f07512c816/.github/workflows/ci-pipeline.yaml#L54-L57

However, it is true that we should be limiting its execution to open PRs.

Reply:
You right I missed that piece of code. Thanks